tests/CMakeLists.txt

```diff
@@ -204,7 +204,7 @@ if(WITH_PYTHON)
                restarting/from_7.0.0/UpgradeAndBackupRestore-2.toml)
   add_fdb_test(
     TEST_FILES restarting/to_6.3.5/CycleTestRestart-1.txt
-               restarting/to_6.3.5/CycleTestRestart-2.txt)
+               restarting/to_6.3.5/CycleTestRestart-2.txt IGNORE)
```

Comment:
What's the plan for re-enabling this?

Reply:
I will open a PR on 6.3 to read xxhash3 checksums, then downgrade tests should pass again, and the test can be reenabled.